From 6108af50a9378aff496ab2138794ed748485dd72 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 17 Sep 2018 19:08:21 +0300
Subject: [PATCH] Goya UART CLK: update according to new PLL div

changed div to PCI[div1] to be 16.
It means that UART CLK is 860M/(16+1) =~ 50.588MHz
---

diff --git a/board/habanalabs/goya/uboot_asic.fragment b/board/habanalabs/goya/uboot_asic.fragment
index 0fcf240..c9d383c 100644
--- a/board/habanalabs/goya/uboot_asic.fragment
+++ b/board/habanalabs/goya/uboot_asic.fragment
@@ -2,7 +2,7 @@
 CONFIG_CONARGS="serial0:115200n8"
 CONFIG_BAUDRATE=115200
 CONFIG_BOOTDELAY=0
-CONFIG_DEBUG_UART_CLOCK=107500000
+CONFIG_DEBUG_UART_CLOCK=50588000
 CONFIG_BOOTCOMMAND="getfit && bootm $fileaddr#conf@1"
 CONFIG_DEFAULT_DEVICE_TREE="goya_asic"
 CONFIG_SPI_FLASH_MACRONIX=y
